intellij/src/org/scalafmt/intellij/ScalafmtAction.scala

```diff
@@ -112,13 +112,16 @@ class ScalafmtAction extends AnAction {
       globalConfig = getConfigFileInPath(homeDir)
       configFile <- localConfig.orElse(globalConfig)
       config <- {
-        val x = StyleCache.getStyleForFile(configFile)
-        if (x.isEmpty) {
-          displayMessage(event,
-                         s"Failed to read $configFile",
-                         MessageType.WARNING)
+        val x = StyleCache.getStyleForFileOrError(configFile)
+        x match {
+          case Left(e:Throwable) => {
+            displayMessage(event,
+              s"Failed to read $configFile" + e.getMessage,
```

Comment:
yeah sure should i add -> punctuation for separate config file and error message? Sorry i dont know and understand how to run plugin because as you mention to run it with command Run > Run 'Plugin' but its not work its give error command not found. If you help me to run the plugin i can send you the screenshot also